development/build/styles.js

```diff
@@ -64,18 +64,25 @@ function createStyleTasks({ livereload }) {
 }
 
 async function buildScssPipeline(src, dest, devMode) {
-  if (!sass) {
-    // use our own compiler which runs sass in its own process
-    // in order to not pollute the intrinsics
-    // eslint-disable-next-line node/global-require
-    sass = require('gulp-sass')(require('./sass-compiler'));
-  }
   await pump(
     ...[
       // pre-process
       gulp.src(src),
       devMode && sourcemaps.init(),
-      sass().on('error', sass.logError),
+      gulpSass({
+        // The order of includePaths is important; prefer our own
+        // folders over `node_modules`
+        includePaths: [
+          // enables shortcuts to `@use design-system`, `@use utilities`, etc.
+          './ui/css',
+          './node_modules',
+        ],
+        functions: {
+          // Tell sass where to find the font-awesome font files
+          // update this location in static.js if it changes
+          'faFontPath()': () => new sass.SassString('./fonts/fontawesome'),
```

Comment:
this will be useful once the webpack-based build system is in place, as that system bundles everything it finds, including fonts, automatically.